config/_default/menus/menus.en.yaml

```diff
@@ -3293,6 +3293,11 @@ main:
     parent: cspm
     identifier: resource_catalog
     weight: 114
+  - name: Resource Evaluation Filters
```

Comment:
What's the reason for including this guide in the left nav? I assume it's important or commonly used? Or the "Misconfigurations" navigation item is used to collect common misconfigurations that might appear in different parts of the docs?